pkg/util/cluster.go

```diff
@@ -426,3 +427,48 @@ func isValidCipherName(str string) bool {
 		slices.Contains(openshiftconfigv1.TLSProfiles[openshiftconfigv1.TLSProfileIntermediateType].Ciphers, str) ||
 		slices.Contains(openshiftconfigv1.TLSProfiles[openshiftconfigv1.TLSProfileModernType].Ciphers, str)
 }
+
+func getNodesCount(cl client.Client, nodesType NodesType) (int, error) {
+	if nodesType == ControlPlaneNodes {
+		masterReq, err := labels.NewRequirement("node-role.kubernetes.io/master", selection.Exists, nil)
+		if err != nil {
+			return -1, err
+		}
+		controlplaneReq, err := labels.NewRequirement("node-role.kubernetes.io/control-plane", selection.Exists, nil)
+		if err != nil {
+			return -1, err
```

Comment:
return zero-value (0)

Reply:
this function was refactored. returning 0 on error.